alan_compiler/src/std/root.ln

```diff
@@ -4992,7 +4992,7 @@ fn lte(a: gvec4i, b: gvec4i) = glte{gvec4i, gvec4b}(a, b);
 fn lte(a: gvec4f, b: gvec4f) = glte{gvec4f, gvec4b}(a, b);
 
 fn ggt{I, O}(a: I, b: I) {
-  let varName = '('.concat(a.varName).concat(' < ').concat(b.varName).concat(')');
+  let varName = '('.concat(a.varName).concat(' > ').concat(b.varName).concat(')');
```

Comment:
No matter what, this fix (and the one below) will be kept, as it confused the crap out of me until I inspected the generated shader code and realized something was very wrong.